tania-prof.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE hmtl>
+<hmtl>
+
+    <head>
+        <link rel="stylesheet" type="text/css" href="h1 {.css">
+        <!-- Add icon library -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+        <title>Tania Yeroman's Personal Profile</title>
+    </head>
+    <main>
+
+        <body>
+            <nav>
+                <div class="navbar">
+                    <a href=#>About</a>
+
+                    <a href=#>Experience</a>
+
+                    <a href=#>Contact</a>
+
+                    <a href=#>Home<a/>
+                </div>
+            </nav>
+            <h1>Tania Yeromiyan</h1>
+            <section class="flexcontainer">
```

Comment:
Naming conventions are important as a project gets bigger and bigger. Class and ID names should explain semantically what the element is/does. If you grew this project any bigger, I would recommend renaming from flexcontainer. 

In HTML and CSS, best practice is to separate words using kebab case such as 
```
<section class="flex-container">
```